src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -325,15 +316,57 @@ const DateForm: React.FC<{
       </RowButton>
       {modal ? (
         <DateInputModalWrapper isOpen={datePickerOpen}>
-          {getDateInput()}
+          <DateInput
+            datePickerOpen={datePickerOpen}
+            setDatePickerOpen={setDatePickerOpen}
+            dateValue={dateValue}
+            setDate={setDate}
+            quote={quote}
+            modal={modal}
+          />
         </DateInputModalWrapper>
       ) : (
-        getDateInput()
+        <DateInput
+          datePickerOpen={datePickerOpen}
+          setDatePickerOpen={setDatePickerOpen}
+          dateValue={dateValue}
+          setDate={setDate}
+          quote={quote}
+        />
       )}
     </RowButtonWrapper>
   )
 }
 
+type DateInputProps = {
+  datePickerOpen: boolean
+  setDatePickerOpen: (isOpen: boolean) => void
+  dateValue: Date | null
+  setDate: (date: Date | null) => void
+  quote: OfferQuote
+  modal?: boolean
+}
+
+const DateInput: React.FC<DateInputProps> = ({
+  datePickerOpen,
+  setDatePickerOpen,
+  dateValue,
+  setDate,
+  quote,
+  modal,
+}) => {
+  return (
+    <StyledDateInput
+      open={datePickerOpen}
+      setOpen={setDatePickerOpen}
+      date={dateValue || new Date()}
+      setDate={setDate}
+      hasCurrentInsurer={hasCurrentInsurer(quote)}
+      modal={Boolean(modal)}
+    />
+  )
+}
```

Comment:
nit: This seems so simple we don't really need the abstraction -- could just use `StyledDateInput` directly inside `DateForm`